app/build.gradle

```diff
@@ -58,4 +58,12 @@ dependencies {
 
     // LiveData + ViewModel
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
+
+    // Navigation
+    implementation "androidx.navigation:navigation-fragment-ktx:$version_navigation"
+    implementation "androidx.navigation:navigation-ui-ktx:$version_navigation"
```

Comment:
Don't use legacy dependencies use only AndroidX instead of support-v4

Reply:
👍